server/repository/src/db_diesel/currency.rs

```diff
@@ -92,6 +93,12 @@ fn create_filtered_query(filter: Option<CurrencyFilter>) -> BoxedLogQuery {
             Some(false) => query.filter(currency_dsl::is_home_currency.eq(false)),
             None => query,
         };
+
+        query = match filter.is_active {
+            Some(true) => query.filter(currency_dsl::is_active.eq(true)),
+            Some(false) => query.filter(currency_dsl::is_active.eq(false)),
+            None => query,
+        };
```

Comment:
Fun rust having 500 ways to do one thing: 

```rust
if let Some(tis_active) = filter.is_active {
    query = query.filter(currency_dsl::is_active.eq(tis_active))
}
```

(`tis_active` because for some reason I had imported the field struct `is_active` from clinician and it was colliding - LSP autocomplete throwing me curve balls lol. Anyway I like `tis` lol.)